src/cli/cli.cpp

```diff
@@ -1994,6 +1994,7 @@ void Interpreter::HandleIcmpReceive(Message &               aMessage,
 {
     uint32_t timestamp = 0;
 
+    VerifyOrExit(mServer != NULL);
```

Comment:
It's probably more appropriate to only handle ICMPv6 Echo Reply messages if an ICMPv6 Echo Request message was sent.

Reply:
I see. Make sense (and more correct this way). I went for quick fix for this.
Let me close this PR.